docs/en_US/CONTRIBUTING.md

```diff
@@ -33,10 +33,33 @@ $ git fetch upstream
 $ git checkout -b <my-branch> upstream/master
 ```
 
+### Package Import Specification
```

Comment:
Can we describe all lint and give a lint rule link? Import order is just one of the rule.

Reply:
Done